cli/src/plugins/javadoc/index.ts

```diff
@@ -936,27 +920,95 @@ const makeElementDetailBody: MakeBodyFunction = (args) => {
     .flat(1);
 };
 
+function uniqueify(s: string[]): string[] {
+  return [
+    ...new Set<string>(
+      s.filter((str) => str !== undefined && str !== null && str.trim() !== "")
```

Comment:
For future reference you can just `!= null` to catch undefined OR null

Reply:
If it's possible to have null or undefined, the parameter type should be `(string | null | undefined)[]`